src/monitor/monitor.c

```diff
@@ -405,17 +405,34 @@ static int add_svc_conn_spy(struct mt_svc *svc)
 
 static void svc_child_info(struct mt_svc *svc, int wait_status)
 {
+    int exit_code = 0;
+    int pid = svc->pid;
+    const char *name = (svc->name ? svc->name : "");
```

Comment:
If brackets are needed here? Or they are just for better visibility?

Reply:
That's my coding style preference. 

I don't recall any item in the SSSD coding style that would contradict this.

Reply:
OK